src/views/virtualmachines/actions/components/VirtualMachineMigration/constants.ts

```diff
@@ -0,0 +1,84 @@
+import {
+  K8sModel,
+  K8sResourceCommon,
+  K8sResourceCondition,
+} from '@openshift-console/dynamic-plugin-sdk';
+
+export type MigPlan = K8sResourceCommon & { spec: any; status?: { suffix: string } };
```

Comment:
why dont have this resource and we can generate new version of the package?

Reply:
I don't know really. I can generate it in the kubevirt-api repo and then update it here in another pr.
It comes from openshift and not kubernetes or kubevirt

Reply:
openshift do not seems to have swagger where we can fetch definitions and generate types